src/Fizzics/toolbox.js

```diff
@@ -20,7 +20,13 @@ var FizzicsToolbox = GObject.registerClass(class FizzicsToolbox extends Toolbox
         this.show_all();
 
         this.connect('reset', () => {
-            this._model.reset();
+            const data = new GLib.Variant('a{sv}');
+            const action = Gio.DBusActionGroup.get(
```

Comment:
I think an easier way to call an action remotely might be this:
```js
Gio.DBus.session.call_sync('com.endlessm.Fizzics',
    '/com/endlessm/Fizzics',
    'org.gtk.Actions', 'Activate',
    new GLib.Variant('(sava{sv})', ['reset', [], {}]),
    null, Gio.DBusCallFlags.NONE, -1, null);
```
or the async variant thereof (remember that `Gio.DBus.session` hides a sync operation)